api_docs.yml

```diff
@@ -3732,7 +3732,7 @@ paths:
           type: string
 
       responses:
-        200:
+        201:
```

Comment:
are we creating a new resource here @Mubangizi?

Reply:
Its a post request, so it shouldnt be a 200. And thats in the docs